cinema-soft/src/test/java/softserve/academy/cinemasoft/service/CategoryServiceImplTest.java

```diff
@@ -52,6 +52,7 @@ public void testRemoveCategory() {
         categoryService.removeCategory(anyInt());
 
         verify(categoryRepository, only()).deleteById(anyInt());
```

Comment:
You should specify a number:
    categoryService.removeCategory(1);
    verify(categoryRepository, only()).deleteById(1);

Reply:
Ok.

Reply:
Not fixed ?

Reply:
Fixed now.